zookeeper-server/src/test/java/org/apache/zookeeper/ZKUtilTest.java

```diff
@@ -85,4 +90,13 @@ public void testUnreadableFileInput() throws Exception {
         assertEquals(expectedMessage, error);
     }
 
+    @Test
+    public void testListRootPathSuccess() throws IOException, InterruptedException, KeeperException {
+        TestableZooKeeper zk = createClient();
+        zk.setData("/", "some".getBytes(), -1);
+        zk.create("/a", "some".getBytes(), ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
+        zk.create("/a/b", "some".getBytes(), ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
+        List<String> list = ZKUtil.listSubTreeBFS(zk, "/");
+        assertIterableEquals(list, Arrays.asList("/a", "/a/b"));
+    }
```

Comment:
the unit case is not sufficient. e.g.,:
- we should assert the size/content of the list. 
- we also need to test(`listSubTreeBFS`) the path: `/a,`  `/a/b`,  other than only` /`

Reply:
OK, I'll fix it.